packages/blockchain-tools/src/interfaces/XrplClient.ts

```diff
@@ -0,0 +1,39 @@
+import { type Payment, Client } from "xrpl"
+import XrplCommon from "./XrplCommon"
+
+export default class XrplClient extends XrplCommon {
+
+  async sendPayment({ address, amount, memo }: { address: string; amount: number; memo?: string }) {
```

Comment:
This looks just about the same as the XrplCommon `sendPayment` function, is there a difference? If no, then we can get rid of this function.

Reply:
Yes we can get rid of it, that's duplicate code